test_scripts/Smoke/HeartBeat/ATF_HeartBeat_App_does_not_send_HB_and_does_not_respond.lua

```diff
@@ -115,4 +117,4 @@ function Test.Stop_SDL()
   StopSDL()
 end
 
-return Test
\ No newline at end of file
+return Test
```

Comment:
Smoke test changes seem out of scope, any reason they are included in this PR?

Reply:
@jacobkeeler Actually changes for smoke should not be shown by GitHub since they are already in develop. For example please take for these 2 versions of ```ATF_HeartBeat_App_does_not_send_HB_and_does_not_respond.lua``` script:
[develop](https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/develop/test_scripts/Smoke/HeartBeat/ATF_HeartBeat_App_does_not_send_HB_and_does_not_respond.lua)
[feature](https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/feature/iap2_transport_switch_tests/test_scripts/Smoke/HeartBeat/ATF_HeartBeat_App_does_not_send_HB_and_does_not_respond.lua) 

They are identical.

This issue can be resolved by re-basing feature branch on current develop.
If you don't mind I'll do this.